podcasts/Sharing/ShareDestination.swift

```diff
@@ -62,7 +76,8 @@ struct ShareDestination: Hashable {
                 return
             }
 
-            let pasteboardItems = [["com.instagram.sharedSticker.backgroundImage": backgroundImage,
+            let dataKey = type == .mpeg4Movie ? "com.instagram.sharedSticker.backgroundVideo" : "com.instagram.sharedSticker.backgroundImage"
+            let pasteboardItems = [[dataKey: data,
                                     "com.instagram.sharedSticker.contentURL": attributionURL]]
             let pasteboardOptions: [UIPasteboard.OptionsKey: Any] = [.expirationDate: Date().addingTimeInterval(60 * 5)]
```

Comment:
Nit: Could we define a constant for the 60*5, something like ```.fiveMinutes```

Reply:
We actually have an extension for this which I totally forgot about.

Changed to `5.minutes`